### ISSUE TITLE
3 messages and stop responding

### ISSUE THREAD

**Issue by ustedestaaqui** [PawanOsman/ChatGPT#99]
3 messages and stop responding
3 messages and it stops responding and I have to restart the server, I think it is bing's limitation

**Comment by PawanOsman** [comment]
@ustedestaaqui Bing? How is Bing related to this?

**Comment by ustedestaaqui** [comment]
> @ustedestaaqui¿Bing? ¿Cómo se relaciona Bing con esto?

Haha so Openai, I thought it was bing, my previous reverse engineering api code was with chatgpt through bing

**Comment by PawanOsman** [comment]
@ustedestaaqui, do you send multiple requests at the same time or within a short period?

**Comment by ustedestaaqui** [comment]
No, normal, like a conversation, he answers me 2 or 3 times and then no more. and I have to restart the host

**Comment by PawanOsman** [comment]
instead of restarting it wait for 2-3 minutes and see if you can send message again

**Comment by ustedestaaqui** [comment]
> en lugar de reiniciarlo, espere de 2 a 3 minutos y vea si puede enviar el mensaje nuevamente

thank you
